src/Umbraco.Web.UI.Client/src/packages/core/components/table/table.element.ts

```diff
@@ -274,18 +271,17 @@ export class UmbTableElement extends UmbLitElement {
 
 	private _renderHeaderCheckboxCell() {
 		if (this.config.hideIcon && !this.config.allowSelection) return;
-
 		return html`
 			<uui-table-head-cell style="--uui-table-cell-padding: 0; text-align: center;">
 				${when(
 					this.config.allowSelection,
-					() =>
-						html` <uui-checkbox
-							label="Select All"
+					() => html`
+						<uui-checkbox
+							aria-label="Select all rows"
```

Comment:
Would "Select all items" be better since it is actually entities being selected?

Reply:
I have updated the word to use the localized version of "select all".